lang/en.ts

```diff
@@ -907,6 +907,90 @@ export const commands = {
     selectLink: 'Select a link to open',
   },
   project: {
+    profile: {
+      describe: 'Commands for managing project profiles',
+      verboseDescribe: `Commands for managing project profiles\n\nProfiles are stored at the root of your project's source directory and they make configuration dynamic. Use them to couple specialized configurations of your project to specific HubSpot accounts.\n\nRun ${uiCommandReference('hs project profile add')} to get started!`,
+      add: {
+        describe: 'Add a new project profile',
+        verboseDescribe: `Add a new project profile\n\nProfiles enable you to reference variables in your component configuration files. Use the syntax ${chalk.bold('\${VARIABLE_NAME}')} to reference profile variables in your component configuration files. Then target the profile using the ${uiCommandReference('--profile')} flag when you upload your project.`,
+        example: 'Add a new project profile named hsprofile.qa.json',
+        logs: {
+          copyExistingProfile: (profileName: string) =>
+            `Found an existing profile. We can copy the variables from ${chalk.bold(
+              profileName
+            )} into your new profile.`,
+          copyExistingProfiles:
+            'Found existing project profiles. We can copy the variables from one of them into your new profile.',
+          profileAdded: (profileName: string) =>
+            `Project profile ${chalk.bold(profileName)} was successfully added`,
+        },
+        prompts: {
+          namePrompt: '[name] Enter a name for the new project profile: ',
+          emptyName: 'Profile name cannot be empty',
+          targetAccountPrompt:
+            '[target-account] Select a target account for this profile',
+          copyExistingProfilePrompt: 'Select a profile to copy variables from',
+          copyExistingProfilePromptEmpty: "Skip (don't copy any variables)",
+          invalidProfileName:
+            'Profile name cannot contain special characters or spaces',
+        },
+        errors: {
+          noProjectConfig:
+            'No project config found. Please run this command from a project directory.',
+          profileExists: (profileName: string) =>
+            `Profile ${chalk.bold(profileName)} already exists. Please choose a different name.`,
+          invalidTargetAccount: 'Target account is not configured in the CLI',
+          noAccountsConfigured: 'No accounts configured in the CLI',
+          failedToLoadProfile: (profileName: string) =>
+            `Unable to copy variables. Failed to load profile ${chalk.bold(profileName)}`,
+          failedToCreateProfile: 'Failed to create profile',
+        },
+        positionals: {
+          name: 'The name of the project profile',
+        },
+        options: {
+          targetAccount: 'The target account ID for this profile',
+        },
+      },
+      remove: {
+        describe: 'Remove an exisiting project profile',
```

Comment:
Typo

Reply:
Thanks for catching this one 👌 fixed in [9494987](https://github.com/HubSpot/hubspot-cli/pull/1520/commits/94949878726ed9bab4a22eb036076b79a2817582)